mojo/stdlib/test/utils/test_numerics.mojo

```diff
@@ -170,6 +170,36 @@ def test_max_finite():
     assert_true(overflow_fp[DType.float32]())
     assert_true(overflow_fp[DType.float64]())
 
+    assert_equal(max_finite[DType.int8](), 127)
+    assert_equal(max_finite[DType.uint8](), 255)
+    assert_equal(max_finite[DType.int16](), 32767)
+    assert_equal(max_finite[DType.uint16](), 65535)
+    assert_equal(max_finite[DType.int32](), 2147483647)
+    assert_equal(max_finite[DType.uint32](), 4294967295)
+    assert_equal(max_finite[DType.int64](), 9223372036854775807)
+    assert_equal(max_finite[DType.uint64](), 18446744073709551615)
+    # FIXME(#5214): uncomment once it is closed
+    # assert_equal(
+    #     max_finite[DType.int128](), 0x7FFF_FFFF_FFFF_FFFF_FFFF_FFFF_FFFF_FFFF
```

Comment:
Nit: I'm not sure I find the hex representation strictly better here. At least, better enough that these comments and the FIXME is worth it.